pygmt/helpers/caching.py

```diff
@@ -14,6 +14,7 @@ def cache_data():
         # List of GMT remote datasets.
         "@earth_age_01d_g",
         "@earth_day_01d",
+        "@earth_dist_01d",
```

Comment:
Also need to add `@N00W030.earth_dist_01m_g.nc` near line 48.

Reply:
Add it in commit https://github.com/GenericMappingTools/pygmt/pull/3706/commits/2404d9c0f2bd78c2c189553a34956da044cc5377.